Cargo.toml

```diff
@@ -26,10 +26,6 @@ branch = "master"
 git = "https://github.com/radicle-dev/radicle-link"
 branch = "master"
 
-[patch.crates-io.ethers]
-git = "https://github.com/gakonst/ethers-rs"
-branch = "master"
-
```

Comment:
To allow using a git dependency in the `reward` CLI, the only way to solve it was to remove this patch.
Will see if I find a better way to make this work since this eventually brakes other CLIs in this repo..

Reply:
After talking with @Ryanmtate I think we should avoid patching ethers in the global Cargo.toml and let each tool define ethers for itself.

Reply:
Eventually good to standardize the client-tools on a single version of ethers-rs, otherwise will have conflicting versions as a dependency.

Maybe instead we do include the patch, but on the `rev` you need.

Reply:
never mind I was able to fix the conflicts I had with the latest commits on the ethers master branch so I brought the patch back